src/app/Instance/BridgeOverview/BridgeOverview.test.tsx

```diff
@@ -156,15 +159,40 @@ describe("Bridge Overview", () => {
     expect(onDeleteProcessor).toHaveBeenCalledTimes(1);
   });
 
+  it("should disable the delete button, when processor is not in ready or failed state", async () => {
+    const { comp } = setupBridgeOverview({
+      processorList: [
+        {
+          kind: "Processor",
+          id: "a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7",
+          name: "Processor one",
+          href: "/api/smartevents_mgmt/v2/bridges/3543edaa-1851-4ad7-96be-ebde7d20d717/processors/a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7",
+          submitted_at: "2022-04-12T12:10:46.029400+0000",
+          published_at: "2022-04-12T12:12:52.416527+0000",
+          status: ManagedResourceStatus.Accepted,
+          flows: [],
+          owner: "",
+        },
+      ],
+      bridgeStatus: ManagedResourceStatus.Ready,
```

Comment:
You could eventually reuse the `processor` constant declared on top of the file here and change the status as you need. Same thing for the other test on the edit button.